### DISCUSSION TITLE
Passing Auto Column Data With Automate

### DISCUSSION THREAD

**Discussion by doylegrant** [Budibase/budibase#1836] [Ideas]
Is there away to pass the data of a formula or auto column from Table A to Table B with automate?

Basically would like to use Automate to create a `linked row` or even just something referencing a row in `Table_B` to `Table_A` when a new row in `Table_A` is created.

I am able to pass `{{ trigger.row.enteredData }}` but `{{ trigger.row.formulaCol }}` or `{{ trigger.row.AutoID }}` is empty.

**Comment by mjashanks** [comment]
Unfortunately not at the moment - there are some column type that automations don't currently support. We really need to fix this - automations are due some attention soon. 

Thanks for reporting, sorry I couldn't give a better answer.

**Comment by eyduh** [comment]
I second this! I have set up a public form and would like to add the AutoID to the confirmation email I have sent with automation at the end.

I might try and setup some workaround using a second table/database, creating rows in the other table then fetching that value.
